src/modules/createNewScore.js

```diff
@@ -0,0 +1,13 @@
+import { postNewScore } from './fetchingFunction.js';
+
+export default function createNewScore() {
+  const userInput = document.querySelector('#user-input');
+  const scoreInput = document.querySelector('#score-input');
+
+  postNewScore({ user: userInput.value, score: Number(scoreInput.value) }).then((resp) => {
+    if (resp.status === 201) {
+      userInput.value = '';
+      scoreInput.value = '';
+    }
+  });
+}
```

Comment:
As part of the project requirements, you are supposed to use the arrow functions instead of the function keyword. Kindly change this and all other function to use the arrow functions. :pray: